### DISCUSSION TITLE
Support for L4S?

### DISCUSSION THREAD

**Discussion by a-nawel** [srsran/srsRAN_Project#698] [General Help] [answered]
Hello, 
I was wondering if support for L4S has already been implemented, and if not, are there any plans?
Thanks!

**Comment by brendan-mcauliffe** [comment, marked as answer]
Hi @a-nawel, 

srsRAN Project doesn't explicitly support L4S, currently I cannot say when/if it will be implemented. It does however support configurations for high-throughput-low-latency scenarios. You can take a look at [this post](https://github.com/srsran/srsRAN_Project/discussions/668) and the associated video for an idea of how to go about configuring srsRAN Project for these scenarios.
